Rakefile

```diff
@@ -115,7 +115,11 @@ task :build => [
 task :install => :build do |t|
   mkdir_p "#{DATADIR}/foreman-installer"
   cp_r Dir.glob('{checks,config,hooks,VERSION,README.md,LICENSE}'), "#{DATADIR}/foreman-installer"
+
   cp "#{BUILDDIR}/foreman-hiera.conf", "#{DATADIR}/foreman-installer/config/foreman-hiera.conf"
+  mkdir_p "#{SYSCONFDIR}/foreman-installer"
+  ln_s "#{DATADIR}/foreman-installer/config/foreman.hiera/custom.yaml", "#{SYSCONFDIR}/foreman-installer/custom-hiera.yaml"
```

Comment:
I think this symlink's backwards, is the file meant to be stored in /etc but the symlink in /usr? It's currently in /usr but the symlink is in /etc.